src/renderer/page/backup/view.jsx

```diff
@@ -37,9 +38,21 @@ class BackupPage extends React.PureComponent<Props> {
                 </p>
                 <p>
                   {__(
-                    'However, it is fairly easy to back up manually. To backup your wallet, make a copy of the folder listed below:'
+                    'However, it is fairly easy to back up manually. To backup your wallet, click the button below to open your wallet directory, and copy the files to a safe location.'
                   )}
                 </p>
+				<div className="walletbackup__actions">
+                <Button
+                  button="primary"
+                  label={__('Open Wallet Directory')}
+                  onClick={() => shell.showItemInFolder(lbryumWalletDir)}
+                />
+                </div>
+		          <p>
```

Comment:
Not sure why the formatting is off for this `<p>` tag. [Prettier](https://github.com/prettier/prettier) should run automatically when you commit.